ISA-XLSX.md

```diff
@@ -745,6 +754,107 @@ For example, the `RUN PERFORMERS` section of an ISA-XLSX `isa.run.xlsx` file may
+|-------|----------|-------------|
+| Workflow Identifier          | String | A mandatory unique identifier, either a temporary identifier supplied by users or one generated by a repository or other database. For example, it could be an identifier complying with the LSID specification. A value MUST be given for this label. |
+| Workflow Title               | String                               | A concise phrase used to encapsulate the purpose and goal of the workflow.  |
+| Workflow Description         | String | A textual description of the workflow, with components such as objective or goals. |
+| Workflow Subworkflow Identifiers | String | Identifiers of the Workflows who're orchestrated in the context of this workflow. Multiple values can be provided by using a semicolon (“;”) Unicode (U0003+B) as a separator (e.g.: Imputation;Testing;Plotting). |
+| Workflow Type                                  | String     | Term to classify the protocol. The term can be free text or from, for example, a controlled vocabulary or an ontology. If the latter source is used the Term Accession Number and Term Source REF fields below are required. |
+| Workflow Type Term Accession Number            | String     | The accession number from the Term Source associated with the selected term. |
+| Workflow Type Term Source REF                  | String     | Identifies the controlled vocabulary or ontology that this term comes from. The Source REF has to match one of the Term Source Name declared in the Ontology Source Reference section. |
+| Workflow URI                                   | String     | Pointer to protocol resources external to the ISA-Tab that can be accessed by their Uniform Resource Identifier (URI). |
+| Workflow Version                               | String     | An identifier for the version to ensure protocol tracking. |
+| Workflow Parameters Name                       | String     | A semicolon-delimited (“;”) list of parameter names, used as an identifier within the ISA-XLSX document. These names are used in the Workflow to list the values used for each protocol parameter. Refer to section Multiple values fields in the Investigation File on how to encode multiple values in one field and match term sources |
+| Workflow Parameters Term Accession Number      | String     | The accession number from the Term Source associated with the selected term. |
+| Workflow Parameters Term Source REF            | String     | Identifies the controlled vocabulary or ontology that this term comes from. The Source REF has to match one of the Term Source Name declared in the Ontology Source Reference section. |
+| Workflow Components Name                       | String     | A semicolon-delimited (“;”) list of a protocol’s components; e.g. instrument names, software names, and reagents names. Refer to section Multiple values fields in the Investigation File on how to encode multiple components in one field and match term sources. |
```

Comment:
> list of a protocol’s components

-->

> list of a workflow’s components

?

Also, suggesting reagent names as an example seems wrong here

Reply:
> > list of a protocol’s components
> 
> -->
> 
> > list of a workflow’s components
> 
> ?
> 
> Also, suggesting reagent names as an example seems wrong here

Yep, this would actually be a good use case for [LabProtocol: ComputationalTool](https://github.com/nfdi4plants/isa-ro-crate-profile/blob/release/profile/isa_ro_crate.md#labprotocol)